app/models/partners/family_request.rb

```diff
@@ -0,0 +1,26 @@
+module Partners
+  class FamilyRequest
+    include ActiveModel::Model
+
+    attr_accessor :comments, :partner
+    attr_reader :items
+
+    def initialize(params, partner: nil, initial_items: nil)
+      @items = [Item.new] * initial_items if initial_items
```

Comment:
It looks like this is copied from what was in partner, but I'm generally wary of multiplying an array with an object like this because I think it creates an array with the same object. But I think maybe it doesn't matter here if each `Item` gets saved as a separate row.

Reply:
@albertchae really good points you bring! I totally agree as well.

>But I think maybe it doesn't matter here if each Item gets saved as a separate row.

🤞 hopefully that is the case :)